public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <title>BC Business Registry</title>
+    <title>BC Registries and Online Services</title>
```

Comment:
@severinbeauvais Leave browser tab title as BC Business Registry for now please (we have this on our list to look at in the future).

Reply:
done